packages/bundle/package.json

```
@@ -76,7 +78,7 @@
     "webpack-visualizer-plugin": "^0.1.11"
   },
   "peerDependencies": {
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react": "16.8.6",
+    "react-dom": "16.8.6"
```

We should keep it as `^16.8.6`. We should allow the app to use `16.*`.

This is peer dependencies. That means, it is not installed through `npm ci` or `npm install`. But will give a warning when someone run `npm install botframework-webchat` in an app that does not have `react@16.*`.

```suggestion
    "react-dom": "^16.8.6"
```